.github/workflows/ci.yml

```diff
@@ -44,14 +44,19 @@ jobs:
       - name: Lint
         if: contains(matrix.os, 'ubuntu')
         run: |
-          cargo clippy --locked --release --all-features --all-targets -- -D clippy::all
```

Comment:
Is there a reason besides CI time to not use `--release`?

Reply:
None of the code has conditions based on release or debug, so it's faster to just do debug on the CI.